src/BuildScriptGenerator/PythonVersions.cs

```diff
@@ -4,7 +4,7 @@ namespace Microsoft.Oryx.Common
 {
     public static class PythonVersions
     {
-        public const string PythonBaseTag = "20191018.1";
+        public const string PythonBaseTag = "20191023.4";
```

Comment:
why do we need this?

Reply:
We really do not need this but we put all python constants in one file in `constants.yml` file and so this just gets generated along with other used properties